Sources/channel/ALChannelClientService.m

```diff
@@ -437,6 +437,8 @@ - (void)updateChannelMetaData:(NSNumber *)channelKey
 
 #pragma mark - Channel Sync
 
+NSString *latSyncCallTimeForChannel = @"";
+
 - (void)syncCallForChannel:(NSNumber *)updatedAt
```

Comment:
From where we are getting updatedAt Value?

Reply:
From UserDefaults. We are storing the last update time after the first create or open conversation api call.